_data/toc/sync-service.yaml

```diff
@@ -37,6 +37,48 @@
           path: '/sync-service/latest/admin/troubleshoot/'
         - title: 'FAQ'
           path: '/sync-service/latest/admin/faq/'
+
+# Alfresco Sync Service 3.3
```

Comment:
shouldn't make a reference for 3.4 ?
please see also the same below references

Reply:
See earlier comments